packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -305,7 +305,9 @@ class AzureDevOpsServerManager {
     const workItems = [] as WorkItem[]
     let firstError: Error | undefined
     const uri = `https://${instanceId}/_apis/wit/workitemsbatch?api-version=7.1-preview.1`
-    const payload = !!fields ? {ids: workItemIds, fields: fields} : {ids: workItemIds}
+    const payload = !!fields
+      ? {ids: workItemIds, fields: fields}
+      : {ids: workItemIds, $expand: 'All'}
```

Comment:
The [API doc](https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/work-items/get-work-items-batch?view=azure-devops-rest-6.0#workitemexpand) for the `$expand` field is not helpful, do we need 'All' or would a smaller subset suffice?

Reply:
The documentation lists the values that can be expanded here - https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/work-items/get-work-items-batch?view=azure-devops-rest-6.0#workitemexpand.

We are looking to return the hypermedia links, so we can slightly reduce the size of the returned payload by changing the value from "All" to "Links".  I made that change and tested it.